### PR TITLE
Complete the project

### PR DESCRIPTION
**Pull request by luccamicale**
## Bookstore Components 💯 

##https://github.com/luccamicale/bookstore

## Checklist before requesting a review 👍 
- [x] I have performed a self-review of my code
- [x] I added the components to Bookstore
- [x] I worked with styles modules
- [x] My code is very tidy and clean. 🔢 🥇 
bookstore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,70 @@
...
 
-Give a ⭐️ if you like this project!
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+### Deployment
```
Comment (youmari) [inline README.md]: - [x] it would be better not deleting the microverse template headers and just customize it to your own in order to keep it clean

**Review by youmari — APPROVED** [review]
Hi @luccamicale , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luccamicale** [comment]
I modified my README.md. Now the project is complete!!! 💯 👍 🥇 🔢

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @luccamicale👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**